crates/typed-store/src/rocks/mod.rs

```diff
@@ -1794,6 +1794,23 @@ pub struct DBOptions {
 /// Creates a default RocksDB option, to be used when RocksDB option is not specified..
 pub fn default_db_options() -> DBOptions {
     let mut opt = rocksdb::Options::default();
+
+    // One common issue when running tests on Mac is that the default ulimit is too low,
+    // leading to I/O errors such as "Too many open files". Raising fdlimit to bypass it.
+    if let Some(limit) = fdlimit::raise_fd_limit() {
+        // on windows raise_fd_limit return None
+        opt.set_max_open_files((limit / 8) as i32);
+    }
+
+    let row_cache = rocksdb::Cache::new_lru_cache(300_000).expect("Cache is ok");
+    opt.set_row_cache(&row_cache);
+
+    // The table cache is locked for updates and this determines the number
+    // of shareds, ie 2^10. Increase in case of lock contentions.
+    opt.set_table_cache_num_shard_bits(10);
+
+    opt.set_compression_type(rocksdb::DBCompressionType::None);
```

Comment:
Just curious do we know the impact on disk space if we turn compression off by default vs how much we gain on speed for reads/writes?

Reply:
oh just noticed this is just moved code and this is the behavior we already have, you can ignore this comment.